src/mito2/src/error.rs

```diff
@@ -575,6 +575,9 @@ pub enum Error {
 
     #[snafu(display("Invalid region options, {}", reason))]
     InvalidRegionOptions { reason: String, location: Location },
+
+    #[snafu(display("checksum mismatch (got: {}, wanted: {})", got, wanted))]
```

Comment:
Prefer `actual` and `expected` personally.
```suggestion
    #[snafu(display("checksum mismatch (actual: {}, expected: {})", got, wanted))]
```